event/eventtest/handler.go

```diff
@@ -30,7 +31,7 @@ type EventHandler struct {
 }
 
 // Handle captures the given event and stores it for later assertions
-func (handler *EventHandler) Handle(ctx context.Context, events []*models.SearchDataImportModel) error {
+func (handler *EventHandler) Handle(ctx context.Context, cfg *config.Config, events []*models.SearchDataImportModel) error {
```

Comment:
Why is the cfg been passed into the Handle method and not used?

Reply:
removed now